src/main/scala/io/iohk/ethereum/db/components/StoragesComponent.scala

```diff
@@ -19,6 +19,8 @@ trait StoragesComponent {
 
     val mptNodeStorage: MptNodeStorage
 
+    val nodeStorage: NodeStorage //FIXME This storage is similar to MPTNodesStorage but
```

Comment:
@adamsmo we might, as we have discussed at least this leads to confusion ... + @mcsherrylabs